package.json

```
@@ -47,6 +47,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.22.6",
+    "@storybook/types": "^8.0.0-rc.2",
```

This is probably better

```suggestion
    "@storybook/types": "^8.0.0",
```

Also remember to update the lock file.

---

Done